src/components/FavoritesBtn.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+import { FavoriteMovie } from '../models/index';
+import api from '../service/api';
+
+import '../styles/FavoritesBtn.scss';
+
+const FavoritesBtn: React.FC<FavoriteMovie> = ({ movieId }) => {
+  const [icon, setIcon] = useState(api.isIdInFavorites(movieId));
```

Comment:
What is a type of icon?

Reply:
The type of  icon is  a boolean

Reply:
you can set type for "setState" as useState<boolean>(api.isIdInFavorites(movieId));

Reply:
Thanks, i'll have a look at this one